Makefile

```diff
@@ -9,6 +9,11 @@ dev:
 	poetry install --extras "pydantic"
 	pre-commit install
 
+dev-gitpod:
+	pip install --upgrade pip poetry
```

Comment:
python3 -m pip?

Reply:
One thing to verify is whether additional dependencies will work as expected if anyone does: `poetry add a_library --dev`

Reply:
Hi @heitorlessa.
1 - I think it's better to use pip directly instead of the python3 pip module. The only reason I used python3 -m pip to install pre-commit is because git looks for the system path to python (/home/gitpod...) when running pre-commit hook. To install poetry and extra dependencies, I think it more suitable to use pip directly because it will install dependencies in the user's workspace (/workspace...).

2 - It makes a lot of sense to install a new dependency and test if it's working as expected, good catch. Here is the screenshots installing a new dependency and using this. Please disregard the purpose of this test, it's just to demonstrate that it works.

![image](https://user-images.githubusercontent.com/4295173/182372277-49b5c882-5157-432e-b69a-dc158686982e.png)
![image](https://user-images.githubusercontent.com/4295173/182372468-29042898-4f36-42d5-9ec7-2b7d9dad0a21.png)
![image](https://user-images.githubusercontent.com/4295173/182372537-0589af30-04e5-4365-a4a2-0c2a2bcfbea6.png)
![image](https://user-images.githubusercontent.com/4295173/182372568-7bde1098-7277-4c1e-8c0f-ff716b886fcb.png)
![image](https://user-images.githubusercontent.com/4295173/182372619-5de9b38d-3391-4a08-b24a-451c811d357b.png)
![image](https://user-images.githubusercontent.com/4295173/182372666-9999e37a-4df8-4ee8-991d-1eefdc26e942.png)